lib/DaxMailer/Schema/Result/Subscriber.pm

```diff
@@ -29,6 +29,12 @@ has_many logs => 'DaxMailer::Schema::Result::Subscriber::MailLog' => {
     'foreign.campaign'      => 'self.campaign',
 };
 
+has_many bounce => 'DaxMailer::Schema::Result::Subscriber::Bounce' => {
+    'foreign.email_address' => 'self.email_address',
+} => {
+    join_type     => "LEFT",
+};
```

Comment:
Commit message for this should have been 'soft rel'

Allows us to use `search_related` clauses without generating a FK rel in the schema, which doesn't work for this since the bounce table has non-subscriber addresses.